samples/dashboards/components/custom-threshold-component/demo.js

```diff
@@ -0,0 +1,166 @@
+                    continue;
+                }
+
+                componentOptions = U.merge(componentOptions, threshold.options);
+                if (threshold.component) {
+                    CurrentComponent =
+                        ComponentRegistry.types[threshold.component];
+                }
+            }
+        }
+
+        if (!this.component || this.component !== Component) {
+            this.parentElement.innerHTML = '';
+            this.component =
+                new CurrentComponent(this.cell, componentOptions).load();
+        } else {
+            this.component.update(componentOptions);
+        }
+
+        return this;
+    }
+}
+
+ComponentRegistry.registerComponent('Threshold', ThresholdComponent);
+
+const board = Dashboards.board('container', {
+    gui: {
+        layouts: [{
+            rows: [{
+                cells: [{
+                    id: 'dashboard-col-0'
+                }]
+            }]
+        }]
+    },
+    components: [{
+        type: 'Threshold',
+        cell: 'dashboard-col-0',
+        component: 'HTML',
+        value: 10,
+        options: {
+            title: {
+                text: 'Threshold Component'
+            },
+            elements: [{
+                tagName: 'p',
+                textContent: `This is a demo of a custom threshold component
+                that allows you to adjust your content depending on the values
+                according to the appropriate thresholds. Move the slider above
+                the board and see how the options and components will change.`,
+                style: {
+                    padding: '0 14px',
+                    textAlign: 'justify'
+                }
+            }]
+        },
+        thresholds: [{
+            min: 15,
+            component: 'KPI',
+            options: {
+                title: 'KPI Component',
+                subtitle: 'You can use any component here, e.g. KPI.'
+            }
+        }, {
+            min: 40,
+            max: 69,
+            component: 'Highcharts',
+            options: {
+                title: '',
+                chartOptions: {
+                    title: {
+                        text: 'Highcharts Component'
+                    },
+                    subtitle: {
+                        text: 'You can use also the Highcharts Component.'
+                    },
+                    series: [{
+                        data: [1, 2, 3],
+                        animation: false
+                    }]
+                }
+            }
+        }, {
+            min: 70,
+            options: {
+                subtitle: `Options from previous thresholds, if not limited by
+                the 'min' option or overwritten, are passed to the next
+                thresholds.`
+            }
+        }, {
+            min: 80,
+            options: {
+                style: {
+                    background: 'blue'
```

Comment:
This colour is super bright 😅 can we please use something a bit more pale?

Reply:
I prefer to use hex instead of "names" of colors.